app/setting/controllers/setting-categories-edit-controller.js

```diff
@@ -33,6 +37,12 @@ function (
     $scope.saveTag = function (tag) {
         $scope.processing = true;
         TagEndpoint.update({id: $routeParams.id}, tag, function () {
+            CacheManager.updateCacheItem(
```

Comment:
Can we bump this up into the endpoint itself? there must be some way to catch the responses there..